content/docs/2.9/scalers/etcd.md

```diff
@@ -0,0 +1,117 @@
++++
+title = "Etcd"
+availability = "v2.9+"
+maintainer = "Community"
```

Comment:
See https://github.com/kedacore/keda/pull/3598#issuecomment-1227620866

Reply:
@WindNotStop would be nice if we can list here `Huawei Cloud` or `Huawei` or any other way you like.